homeassistant/components/scrape/config_flow.py

```diff
@@ -157,7 +155,16 @@ def validate_sensor_setup(
     ),
 }
 OPTIONS_FLOW = {
-    "init": SchemaFlowFormStep(DATA_SCHEMA_RESOURCE),
+    "init": SchemaFlowMenuStep(["resource", "add_sensor"]),
+    "resource": SchemaFlowFormStep(
+        DATA_SCHEMA_RESOURCE,
+        validate_user_input=validate_rest_setup,
+    ),
+    "add_sensor": SchemaFlowFormStep(
+        DATA_SCHEMA_SENSOR,
+        suggested_values=lambda _: {},
```

Comment:
Why is this needed? If we don't want any suggested values, should we allow setting `suggested_values` to `None`, with some  sentinel as default value?